docker/ubuntu20/openvino_tensorflow_cgvh_runtime_2.2.0-serving.dockerfile

```diff
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ARG TF_SERVING_VERSION="2.9.0"
```

Comment:
TF_SERVING_VERSION should also be 2.9.2 ?

Reply:
Sure, I will update this. When we released 2.1.0 Dockerfiles, only 2.9.0 TF Serving images were available. Now we can update to 2.9.2 as I see it's available on DockerHub.